dev/try_builders.json

```diff
@@ -532,13 +532,6 @@
       "enabled": true,
       "run_if": ["dev/**", "packages/flutter_tools/**", "bin/**"]
     },
-    {
-      "name": "Mac dart_plugin_registry_test",
-      "repo": "flutter",
-      "task_name": "dart_plugin_registry_test",
-      "enabled": true,
```

Comment:
Another option: you can set `"enabled": false` if you want to reenable it back later.